docs/_functions/domain/CF_WORKER_ROUTE.md

```diff
@@ -0,0 +1,29 @@
+---
+name: CF_WORKER_ROUTE
+parameters:
+  - pattern
+  - script
+---
+
+`CF_WORKER_ROUTE` uses [Cloudflare Workers](https://developers.cloudflare.com/workers/) 
+API to setup [routes](https://developers.cloudflare.com/workers/platform/routes)
+for a given domain.
+
+If _any_ `CF_WORKER_ROUTE` function is used then `dnscontrol` will manage _all_ 
+Worker Routes for the domain.
```

Comment:
If this means it will delete routes that were added via the web portal, be extremely clear:

```
for the domain. To be clear: this means it will delete existing routes that were created outside of DNSControl.
```